src/main/flight/pid.c

```diff
@@ -89,8 +92,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
 {
     RESET_CONFIG(pidProfile_t, pidProfile,
         .pid = {
-            [PID_ROLL] =  { 40, 40, 20 },
-            [PID_PITCH] = { 58, 50, 22 },
+            [PID_ROLL] =  { 44, 40, 33 },
```

Comment:
oops - for a crappy standard frame this might be to high.
i have flown this value.
the modded pid controller doesn't inject too much noise.
the values can be upped quite a bit.